sass/components/dropdown.sass

```diff
@@ -31,6 +31,18 @@ $dropdown-divider-background-color: $border !default
       padding-bottom: $dropdown-content-offset
       padding-top: unset
       top: auto
+  // Colors
+  @each $name, $pair in $colors
+    $color: nth($pair, 1)
+    $color-invert: nth($pair, 2)
+    &.is-#{$name}
+      .dropdown-trigger
+        .button
+          @extend .is-#{$name}
```

Comment:
What is this extending??

Reply:
It extends every color class (e.g. `is-primary`), but I just saw that it actually extends those classes of *every* element/component, so it leads to unclear CSS output. It's tricky to implement this part. It needs work, I'll be thinking in a better approach to this.